actions.go

```diff
@@ -66,60 +112,21 @@ var (
 		security.EmptyPermission,
 		"\nJe te dis ce que j'ai dans mon ventre !",
```

Comment:
It would be interesting to have smarter help message that provides an example for using the action by extrapolating the possible parameters of the action.

Reply:
please make a ticket out of that idea

Reply:
Ticket #43 created